triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -62,14 +59,21 @@ case class LedgerConfig(
     commandTtl: Duration,
 )
 
-class Server(dar: Option[Dar[(PackageId, Package)]]) {
-  private var triggers: Map[UUID, TriggerRunnerWithToken] = Map.empty;
+final case class RunningTrigger(
+    triggerId: UUID,
```

Comment:
The names here seem a bit confusing to me. The first identifies a running trigger. The second identifies the code of a trigger which can run 0-many times.

Reply:
I think I'm in agreement with this @rohanjr. Perhaps `triggerScriptIdent`?

Reply:
How about `triggerSourceId`?

Reply:
The UUID would be more accurately named `runningTriggerId` but that might be unwieldy as it's used a lot.

Reply:
Maybe something like `triggerInstance` and `triggerName` to make it clear that one is the static name and the other is an instance of that?